pandas/tests/io/test_fsspec.py

```diff
@@ -95,21 +95,19 @@ def test_to_csv_fsspec_object(cleared_fs, binary_mode, df1):
 
     path = "memory://test/test.csv"
     mode = "wb" if binary_mode else "w"
-    fsspec_object = fsspec.open(path, mode=mode).open()
-
-    df1.to_csv(fsspec_object, index=True)
-    assert not fsspec_object.closed
+    with fsspec.open(path, mode=mode).open() as fsspec_object:
+        df1.to_csv(fsspec_object, index=True)
+        assert not fsspec_object.closed
     fsspec_object.close()
```

Comment:
These `fsspec_object.close()` shouldn't be needed with the context manager now? (Same as below)

Reply:
Good catch, fixed :)